docs/src/content/docs/next/guides/on_song_change.mdx

```diff
@@ -130,3 +130,43 @@ for the script to execute and increment the play count.
         ```
 
 </Steps>
+
+## Automatically fetch lyrics
+
+Provided by [wvffle on github](https://github.com/mierak/rmpc/issues/230)
+
+```bash
+#!/usr/bin/env bash
+
+if [ "$HAS_LRC" = "false" ]; then
+    LRCLIB_INSTANCE="https://lrclib.net"
+    LYRICS_DIR="$HOME/Music/lyrics"
```

Comment:
I'd move those two above the condition, so that people could see those as configuration vars